packages/e2e/features/ui/components/authenticator/confirm-sign-up.feature

```diff
@@ -54,11 +52,11 @@ Feature: Confirm Sign Up
     And I confirm my password
     And I click the "Create Account" button
     Then I see "Confirmation Code"
-    # Mocking these two calls is much easier than intercepting 6+ network calls with tokens that are validated & expire within the hour
```

Comment:
I'm totally fine with removing this comment but should we?

Reply:
I'll add it back in